apps/studio/components/interfaces/Settings/Database/BannedIPs.tsx

```diff
@@ -80,19 +90,26 @@ const BannedIPs = () => {
       </div>
       <FormPanel>
         {ipList && ipList.banned_ipv4_addresses.length > 0 ? (
-          ipList.banned_ipv4_addresses.map((ip) => (
-            <div key={ip} className="px-8 py-4 flex items-center justify-between">
-              <div className="flex items-center space-x-5">
-                <IconGlobe size={16} className="text-foreground-lighter" />
-                <p className="text-sm font-mono">{ip}</p>
-              </div>
-              <div>
-                <Button type="default" onClick={() => openConfirmationModal(ip)}>
-                  Unban IP
-                </Button>
+          ipList.banned_ipv4_addresses.map((ip) => {
+            return (
+              <div key={ip} className="px-8 py-4 flex items-center justify-between">
+                <div className="flex items-center space-x-5">
+                  <IconGlobe size={16} className="text-foreground-lighter" />
+                  <p className="text-sm font-mono">{ip}</p>
+                  {ip === userIPAddress && (
+                    <span className="text-sm font-bold text-dark-500 bg-gray-300 [[data-theme*=dark]_&]:bg-zinc-900 px-2 py-1 rounded-full">
+                      YOUR IP
```

Comment:
```suggestion
                      Your IP
```

Might look less agressive 🗡️